Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -642,6 +657,16 @@ for (const auto & particle_diag : particle_diags) {
         pc->getCharge(), pc->getMass(),
         isBTD, isLastBTDFlush);
     }
+
+    auto hasOption=m_OpenPMDoptions.find("FlattenSteps");
+    bool flattenSteps = isBTD && (m_Series->backend() == "ADIOS2") && (hasOption != std::string::npos);
+
+    if ( flattenSteps)
```

Comment:
```suggestion
    if (flattenSteps)
```
I would suggest to eliminate this space